napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -113,6 +114,13 @@ def __init__(
         layout = QVBoxLayout()
         layout.addLayout(hlayout2)
         layout.addWidget(self._table)
+        layout.addWidget(
+            QLabel(
+                trans._(
+                    "To clean shortcut use Backspace or Delete. To set one of this as shortcut, first clean previous shortcut."
```

Comment:
I don't love the verb `clean` in this context. Also added how to edit, because I'm not sure visually it's clear that the keybinds can be edited.

```suggestion
                    "To edit, double-click the keybinding. To unbind a key, use Backspace or Delete. To set Backspace or Delete, first unbind."
```

Reply:
@alisterburt @Czaki any input on the wording here?
Another alternative to "clean"—which I really don't like—is "clear"?

Reply:
You need to open a separate PR (this should be fast) to apply these changes as it is merged. My only remark is maybe `To unbind a key` should be `To unbind a shortcut`